### PR TITLE
Realize DPOS node switching and reward distribution

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: common/config/settings/settings.go [hunk 1]
```diff
@@ -498,6 +498,18 @@ func NewSettings() *Settings {
 		ConfigPath:   "CRConfiguration.CRCommitteeStartHeight",
 		ParamName:    "CRCommitteeStartHeight"})
 
+	result.Add(&settingItem{
+		Flag:         cmdcom.CRClaimDPOSNodeStartHeightFlag,
+		DefaultValue: uint32(0),
+		ConfigPath:   "CRConfiguration.CRClaimDPOSNodeStartHeight",
+		ParamName:    "CRCommitteeStartHeight"})
```
Comment (Houshoupei84) [inline common/config/settings/settings.go:505]: should be CRClaimDPOSNodeStartHeight

File: common/config/settings/settings.go [hunk 2]
```diff
@@ -498,6 +498,18 @@ func NewSettings() *Settings {
...
+
+	result.Add(&settingItem{
+		Flag:         cmdcom.CRClaimDPOSNodePeriodFlag,
+		DefaultValue: uint32(0),
+		ConfigPath:   "CRConfiguration.CRClaimDPOSNodePeriod",
+		ParamName:    "CRCommitteeStartHeight"})
```
Comment (Houshoupei84) [inline common/config/settings/settings.go:511]: should be   CRClaimDPOSNodeStartHeight

File: dpos/state/arbitrators.go [hunk 3]
```diff
@@ -463,11 +465,77 @@ func (a *arbitrators) distributeDPOSReward(height uint32,
 	return
 }
 
-func (a *arbitrators) distributeWithNormalArbitratorsV(height uint32, reward common.Fixed64) (
+func (a *arbitrators) distributeWithNormalArbitratorsV2(height uint32, reward common.Fixed64) (
 	map[common.Uint168]common.Fixed64, common.Fixed64, error) {
 	if len(a.currentArbitrators) == 0 {
 		return nil, 0, errors.New("not found arbiters when " +
-			"distributeWithNormalArbitratorsV")
+			"distributeWithNormalArbitratorsV1")
```
Comment (Houshoupei84) [inline dpos/state/arbitrators.go:472]: should be distributeWithNormalArbitratorsV2

File: common/config/settings/settings.go (hunk 1 above)
Comment (RainFallsSilent) [reply common/config/settings/settings.go:505]: fixed

File: common/config/settings/settings.go (hunk 2 above)
Comment (RainFallsSilent) [reply common/config/settings/settings.go:511]: fixed

File: dpos/state/arbitrators.go (hunk 3 above)
Comment (RainFallsSilent) [reply dpos/state/arbitrators.go:472]: the first one is distributeWithNormalArbitratorsV0..

**Pull request merged**